test/validators/movie.test.js

```diff
@@ -8,22 +8,40 @@ describe('movie validator', () => {
 
   describe('title', () => {
 
-    it('is required', () => {
-      const payload = {};
+    it('is less than 255 characters', () => {
+      const payload = { title: 'a'.repeat(260) };
       const result = Joi.validate(payload, MovieValidator);
 
       expect(result.error.details[0].path[0]).to.eql('title');
-      expect(result.error.details[0].type).to.eql('any.required');
+      expect(result.error.details[0].type).to.eql('string.max');
+    });
+
+    it('cannot be with name', () => {
```

Comment:
This is kind of misleading, since it will actually accept `name`.

Reply:
Resolved: Changed test description